wdn/templates_4.1/includes/scriptsandstyles_speedy.shtml

```diff
@@ -0,0 +1,42 @@
+<noscript>
+    <link rel="stylesheet" href="https://cloud.typography.com/7717652/616662/css/fonts.css" />
+    <link rel="stylesheet" href="https://unlcms.unl.edu/wdn/templates_4.1/css/all.css?dep=$DEP_VERSION$" />
+</noscript>
+<style id="wdn-critical-styles">
+    <!--#include virtual="/wdn/templates_4.1/css/critical.css" -->
+</style>
+<script>
+    var WDNRemoveCriticalStyles = function() {
+        document.querySelector('#wdn-critical-styles').remove();
```

Comment:
If not doing a complex selector, it's best to use `document.getElementById` to avoid the overhead of parsing the selector.